model/src/main/java/org/projectnessie/api/TreeApi.java

```diff
@@ -319,12 +336,17 @@ void assignTag(
       @Valid
           @NotNull
           @RequestBody(
-              description = "New tag content",
+              description =
+                  "Reference hash to which 'tagName' shall be assigned to. This must be either a "
+                      + "'Branch' or 'Tag'.",
               content =
                   @Content(
                       mediaType = MediaType.APPLICATION_JSON,
-                      examples = {@ExampleObject(ref = "tagObj")}))
-          Tag tag)
+                      examples = {
+                        @ExampleObject(ref = "branchObj"),
+                        @ExampleObject(ref = "tagObj")
+                      }))
+          Reference assignTo)
```

Comment:
it feels a bit weird that we're in the `assignTag` method but `assignTo` can refer to a Branch or Tag

Reply:
It's that you can assign a tag to any commit you're allowed to see.